architecture/storage-layer.md

```diff
@@ -0,0 +1,70 @@
+summary: 
+toc: false
+---
+
+The Storage Layer of CockroachDB's architecture reads and writes data from disk.
+
+{{site.data.alerts.callout_info}}If you haven't already, we recommend reading the <a href="overview.html">Architecture Overivew</a>.{{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+## Overview
+
+Each CockroachDB node contains at least one `store`, specified when the node starts, which is where the `cockroach` process reads and writes its data on disk.
+
+CockroachDB stores its key-values on disk using RocksDB, which is treated primarily as a black-box API. Internally, each store contains three instance of RocksDB:
+
+- One for the Raft log
+- One for storing temporary Distributed SQL data
+- One for all other data on the node
+
+In addition, there is also a block cache shared amongst all of the stores in a node. These stores in turn have a collection of range replicas. More than one replica for a range will never be placed on the same store or even the same node.
+
+### Interactions with Other Layers
+
+In relationship to other layers in CockroachDB, the Storage Layer:
+
+- Serves successful reads and writes from the Replication Layer.
+
+## Components
+
+### RocksDB
+
+CockroachDB uses RocksDB––an embedded key-value store––to read and write data to disk. You can find more information about it on the [RocksDB Basics GitHub page](https://github.com/facebook/rocksdb/wiki/RocksDB-Basics).
+
+RocksDB integrates really well with CockroachDB for a number of reasons:
+
+- Key-value store, which makes mapping to our key-value layer very simple
+- Atomic write batches and snapshots, which give us a subset of transactions
+
+Efficient storage for the keys is guaranteed by the underlying RocksDB engine by means of prefix compression.
+
+### MVCC
+
+CockroachDB relies heavily on [multi-version concurrency control (MVCC)](https://en.wikipedia.org/wiki/Multiversion_concurrency_control) to process concurrent requests and guarantee consistency. Much of this work is done by using [hybrid logical clock (HLC) timestamps](transaction-layer.html#time-hybrid-logical-clocks) to differentiate between versions of data, track commit timestamps, and identify a value's garbage collection expiration. All of this MVCC data is then stored in RocksDB.
+
+Despite being implemented in the Storage Layer, MVCC values are widely used to enforce consistency in the [Transaction Layer](transaction-layer.html). For example, CockroachDB maintains a [Timestamp Cache](transaction-layer.html#timestamp-cache), which stores the timestamp of the last time that the key was read. If a write operation occurs at a lower timestamp than the largest value in the Read Timestamp Cache, it signifies there’s a potential anomaly and the transaction must be restarted at a later timestamp.
+
+#### Time-Travel
+
+As described in the [SQL:2011 standard](https://en.wikipedia.org/wiki/SQL:2011#Temporal_support), CockroachDB uses MVCC to support time travel queries.
```

Comment:
This intro doesn't logically lead to the clause after the comma. How is `CockroachDB uses MVCC` described in the SQL standard? Can you try to rephase?